src/rpdk/core/project.py

```diff
@@ -432,9 +449,20 @@ def _set_docs_properties(self, propname, prop, proppath):
             template = self.env.get_template("docs-subproperty.md")
             docs_path = self.root / "docs"
 
+            try:
+                object_properties = (
```

Comment:
The `object` is valid is without `properties` or `patternProperties`.  See [SNS schema](https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-sns/blob/master/aws-sns-topicpolicy/aws-sns-topicpolicy.json#L9). 
Could you help to address this?

Reply:
I see! I didn't know that was possible, now I wonder how CDK handles this 🤔
It seems to be that they have it there as a `any` TS type for the L1 and added some types for the L2... Interesting

Anyways, I pushed new rev for the PR